backend/src/programs/views/ProgramView.py

```diff
@@ -98,7 +95,6 @@ def patch(self, request, pk):
             return Response(status=status.HTTP_406_NOT_ACCEPTABLE)
         except AssertionError:
             return Response(status=status.HTTP_406_NOT_ACCEPTABLE)
-
         program.rss_feed_url = new_url
         program.rss_feed_status = new_status
         return Response(status=status.HTTP_201_CREATED,
```

Comment:
Missing `program.save()` before returning the response :)

Reply:
Fixed